test/webgl/context-state.spec.js

```diff
@@ -2,8 +2,8 @@ import {GL} from 'luma.gl';
 import {getParameter, getParameters, setParameters, withParameters, resetParameters} from 'luma.gl';
 import {GL_PARAMETER_DEFAULTS as GL_PARAMETERS} from '../../src/webgl-utils/parameter-access';
 import {
-  GL_PARAMETER_SETTINGS_TWO,
-  GL_PARAMETER_SETTINGS_TWO_ENUM_FUNCTION
+  GL_PARAMETER_SETTINGS_ONE,
```

Comment:
Agree, `PARAMETER` might be too generic, how about `ENUM_STYLE_SETTINGS` and `FUNCTION_STYLE_SETTINGS`.